transform/macros/tooling/drop_snowflake_objects.sql

```diff
@@ -0,0 +1,44 @@
+{# This macro drops every database (along it's schemas), warehouse and role  #}
```

Comment:
@BAntonellini why do we have this macro?  We are just warning when SF has things not in the repo, right?